sdk/containerregistry/azure-containerregistry/tests/test_container_registry_client.py

```diff
@@ -603,9 +604,50 @@ def test_get_misspell_property(self, containerregistry_endpoint):
         assert last_udpated_on == last_updated_on
     
     @acr_preparer()
-    def test_upload_manifest(self, containerregistry_endpoint):
+    def test_upload_oci_manifest(self, containerregistry_endpoint):
         repo = self.get_resource_name("repo")
         manifest = self.create_oci_manifest()
         client = self.create_registry_client(containerregistry_endpoint)
         
         client.upload_manifest(repo, manifest)
+    
+    @acr_preparer()
+    def test_upload_oci_manifest_stream(self, containerregistry_endpoint):
+        repo = self.get_resource_name("repo")
+        manifest = self.create_oci_manifest()
+        manifest_bytes = json.dumps(manifest.serialize()).encode("utf-8")
+        manifest_stream = BytesIO(manifest_bytes)
```

Comment:
Rather than creating a BytesIO this way - maybe a good test to do would be reading the manifest from a file? That would be a more likely customer scenario.

Reply:
+1, this is a good point! We can discuss in arch board review. To make it consistent with other languages(they all consume "stream" directly), I'll leave it as it is.

Reply:
Have discussed in the ACR group, I think
- Creating stream data from a file is pretty trivial, not a burden in Python.
- Consuming IO directly would have more flexibility. It allows customers to call the API after obtaining stream data from somewhere.

So, I would prefer to only support stream as an input data type. Do you still think it's necessary to have separate methods, i.e. upload_manifest_from_file/upload_blob_from_file, that support file path?